pkg/apis/admissionregistration/validation/validation.go

```diff
@@ -150,18 +151,57 @@ func validateRule(rule *admissionregistration.Rule, fldPath *field.Path, allowSu
 	return allErrors
 }
 
+func isAcceptedAdmissionReviewVersion(v string) bool {
+	return v1beta1.SchemeGroupVersion.Version == v
+}
+
+func validateAdmissionReviewVersions(versions []string, requireRecognizedVersion bool, fldPath *field.Path) field.ErrorList {
+	allErrors := field.ErrorList{}
+
+	// Currently only v1beta1 accepted in AdmissionReviewVersions
+	if len(versions) < 1 {
+		allErrors = append(allErrors, field.Required(fldPath, ""))
+	} else {
+		seen := map[string]bool{}
+		hasAcceptedVersion := false
+		for i, v := range versions {
```

Comment:
should probably constrain the version strings to `utilvalidation.IsDNS1035Label` (here and when validating conversionReviewVersions), since that's what we limit APIService versions to:
```
	for _, errString := range utilvalidation.IsDNS1035Label(apiService.Spec.Version) {
		allErrs = append(allErrs, field.Invalid(field.NewPath("spec", "version"), apiService.Spec.Version, errString))
	}
```